lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -31,7 +31,7 @@ class WP_Theme_JSON_6_1 extends WP_Theme_JSON_6_0 {
 	 * @var string[]
 	 */
 	const ELEMENTS = array(
-		'link'    => 'a:not(.wp-element-button)',
+		'link'    => 'a:not(.wp-element-button, .wp-block-button__link)',
```

Comment:
Shouldn't this be like this instead? 🤔 
```suggestion
		'link'    => 'a:not(.wp-element-button):not(.wp-block-button__link)',
```